packages/jest-preset-mc-app/package.json

```diff
@@ -50,11 +50,13 @@
     "unfetch": "4.1.0"
   },
   "devDependencies": {
-    "@testing-library/react": "10.0.4"
+    "@testing-library/react": "10.0.4",
+    "enzyme": "3.11.0",
+    "jest": "26.0.1"
   },
   "peerDependencies": {
-    "@testing-library/react": "9.x || 10.x",
+    "@testing-library/react": ">=9 || >=10",
     "enzyme": "3.x",
-    "jest": "24.x || 25.x"
+    "jest": ">=24 || >=25 || >=26"
```

Comment:
With the recent PR to upgrade Jest #1481, should we still keep v24?

Reply:
We kept 25 as it would be breaking. We kept 25 as it would be breaking 🎁🤢. To me it‘s fine. There is nothing preventing others using old versions. Maybe we can batch it for a breaking change.